src/react-components/button-social.tsx

```diff
@@ -44,6 +44,14 @@ export const ButtonSocial = ({
       >
         <i className={cn(brandClass, buttonSocialIconStyle({ size }))}></i>
         <div className={buttonSocialTitleStyle}>{title}</div>
+        {/* add another hidden icon to the end to center the text */}
+        <i
```

Comment:
Could just be an empty `<i>` (or `<span>` or whatever^^) element to tidy it up.

Reply:
I think you still need to have the icon there since each icon has a different size in width / height even though the font-size is set.

Example with current hiding of actual icon
![image](https://user-images.githubusercontent.com/18033717/192503871-773880d1-3acd-466b-9fc2-bc8cde1457ad.png)

vs empty icon
![image](https://user-images.githubusercontent.com/18033717/192504058-b01515d0-f428-4671-bbed-f6dc1016f701.png)

there is a slight mis-alignment

Reply:
Ah, right. With the comment it's fine then.